specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -430,6 +472,50 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/transitivepeusages": {
+      "get": {
+        "tags": [
+          "TransitivePEUsage"
+        ],
+        "operationId": "TransitivePEUsage_Get",
+        "description": "Gets the current transitive private endpoint usage information as well as limits for AML resources for given subscription and location.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The location for which resource usage is queried.",
+            "pattern": "^[-\\w\\._]+$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TransitivePEUsagesResult"
```

Comment:
Please make this response look closer to the /usages response (a { "value": [] } top level result object at the very least)

Reply:
Hi, Chris, for this /transitivepeusages, it's not going to be an array returned. So it will be very different from /usages(this is more about the compute unit usage, the naming is not perfect, so it's a list because customer can have multiple computes). If the name is confusing, I can change it to /transitivepeusage, is that okay to you?

Reply:
This is a GET on a resource type (transitivepeusages) so it MUST return a response of the form { "value: [] }.  The array can contain 1 value (your usage info) if you only have one piece of information.

Reply:
Got it. I will make the change